cssom-view-1/Overview.bs

```diff
@@ -1070,21 +1070,16 @@ result of running these steps:
         <dt><a>caret range</a>
         <dd>null
     </dl>
-1. Otherwise, return a <a>caret position</a> where the
-    <a>caret range</a> is a collapsed
-    {{Range}} object for the position
-    where the text insertion point indicator would have been inserted when applying the <a>transforms</a> that apply to the descendants of the
-    <a>viewport</a>, and the other properties are set as follows:
-
-    <dl>
-        <dt><a>caret node</a>
-        <dd>The [=range/start node=]
-            of the <a>caret range</a>.
-
-        <dt><a>caret offset</a>
-        <dd>The [=range/start offset=] of
-            the <a>caret range</a>.
-    </dl>
+1. Otherwise:
+
+    1. Let <var>caretRange</var> to be a collapsed {{Range}} object for the position where the text insertion point indicator would have been inserted when applying
```

Comment:
Do browsers really need to use a live range object to implement this algorithm? At least the non-shadow DOM case seems doable with live ranges in Chromium. https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/dom/document.cc;l=1787;drc=9e110c64fee9c391021dc81944912f906e6ba5d5;bpv=1;bpt=1

Reply:
Are you trying to refer to the concept of range instead? https://dom.spec.whatwg.org/#concept-range

Reply:
Yes. live range is not needed in the algorithm. Will update in next iteration.